modules/programs/tmux.nix

```diff
@@ -83,6 +227,15 @@ in
 {
   options = {
     programs.tmux = {
+      activeBorderStyle = mkOption {
+        default = defaultShortcut;
```

Comment:
why the border style is set by default to the shortcut?

Reply:
my fault, will fix